apps/ui/src/views/Space/Editor.vue

```diff
@@ -528,6 +540,25 @@ watchEffect(() => {
             </AppLink>
             to continue.
           </UiAlert>
+          <UiAlert
+            v-else-if="
+              nonPremiumStrategiesList.length && !proposal?.originalProposal
+            "
+            type="error"
+            class="mb-4"
+          >
+            You cannot create proposals. This space is configured with
+            non-premium strategies (<span
```

Comment:
This sentence is wrong, the proposal is configured with premium strategies
```suggestion
            premium strategies (<span
```

Reply:
Do we have a link to this article ? I only found https://help.snapshot.box/en/articles/11568442-migrating-from-delegation-to-with-delegation-strategy but the title may be misleading